example/server.js

```diff
@@ -3,13 +3,15 @@ var webpack = require('webpack')
 var config = require('./webpack.config')
 
 var compiler = webpack(config)
+var port = 8080;
```

Comment:
Can you replace these two lines with `process.env.X || DEFAULT`? Just for convenience